jscomp/test/more_poly_variant_test.js

```diff
@@ -54,10 +54,14 @@ function g(x) {
 }
 
 function f1(param) {
-  if (param === "As") {
-    return "A";
-  } else {
-    return "other";
+  switch (param) {
+    case "A" :
+    case "B" :
+    case "C" :
+        return "other";
+    case "As" :
+        return "A";
```

Comment:
such switch should be converted to if/then
our switch in IR should support multiple predicates natively, i.e (semantic wise)
```
case "A" "B" "C" : 
```